drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -842,7 +872,8 @@ function DragAndDropBlock(runtime, element, configuration) {
 
             initDraggable();
             initDroppable();
-
+            initializeSlider = sliderInitializer;
+            sliderInitializer();
```

Comment:
I guess i am tagged mistakenly here :)

Reply:
Sorrt @msaqib52 !  Was reviewing your PR at the same time.  Will update.

Reply:
@pomegranited Actually We have to initialize bxslider after the complete page load i.e. after js and images have been loaded. It fails mostly when there are images in dragable tiles. So I used this pageload global variable as indicator that page has been loaded and we can safely initialize bxsliser.

https://github.com/stevenwanderski/bxslider-4#step-3-call-the-bxslider
https://stackoverflow.com/questions/48804763/bxslider-not-working-on-first-load-for-chrome

Reply:
Ah ok, I didn't test the draggable tiles with images in, so didn't see that failing.

In that case, I think your original mechanism was clearer -- setting the `pageLoaded` global instead of replacing the `initializeSlider` function pointer.  At least that way, we can follow the process in the debugger.